sycl/test-e2e/ESIMD/matrix_transpose.cpp

```diff
@@ -5,11 +5,9 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// FIXME: Investigate Windows-specific failures
-// REQUIRES: TEMPORARY_DISABLED
-// UNSUPPORTED: cuda || hip
+// REQUIRES: gpu
+// UNSUPPORTED: cuda || hip || esimd_emulator
```

Comment:
Ok, but I see you tried to fix the issue on esimd_emulator, and may know why it fails there.
If so, can you please add TODO: comment here explainings why test fails on emulator?

Reply:
@fineg74 - please address this comment. If you know why emulator fails, please keep a comment with your opinion explaining why it is marked as unsupported.